### ISSUE TITLE
strange rendering of true 3d dataset

### ISSUE THREAD

**Issue by jrjdavidson** [Melown/vts-backend#22]
Hello, 

I created a highly detailed true 3d dataset from drone and handheld photos using photogrammetry ( sfm in Agisoft metashape). It was a pretty extreme model to start off with given the technical challenges, but it worked quite well so I thought I'd try it in vts. I exported as a slpk and converted using `slpk2vts`

The result is quite good except for the odd behaviour in parts. I attached an image, and here is a live link so you can explore:
https://vtsbackend.canterbury.ac.nz/store/resources/tilesets/caveRock/?pos=obj,172.759753,-43.566130,fix,12.29,176.92,-28.76,0.00,20.00,45.00 
![image](https://user-images.githubusercontent.com/46798391/111116027-4a37f000-85ca-11eb-9e70-e81cf32da890.png)

Happy to share the data if you're curious

**Comment by vaclavblazek** [comment]
Hi, it look really strange. It looks like some problem with conversion. Would it be possible to send me the source SLPK file to investigate?

**Comment by jrjdavidson** [comment]
I sent a link to your @melowntech.com address.

**Comment by jrjdavidson** [comment]
Vaclav, just generated another dataset with a similar problem, I wasn't expecting this as i thought it was a relatively simpler geometry.I had also generated a quick test case earlier and had no issues with it. 

here is the problem dataset:  https://vtsbackend.canterbury.ac.nz/store/resources/tilesets/kakanuiDroneGCP
here is the test without problems: https://vtsbackend.canterbury.ac.nz/store/resources/tilesets/kakanuiDroneTest

It appears the problem crops up when there is an overhang?
again, happy to send you the original slpk files

Have you been able to re-produce the image with the other dataset I sent you?

Let me know if there might be another way, i.e. exporting and converting in another format rather than slpk? metashape can export in various formats, see screenshot below

![image](https://user-images.githubusercontent.com/46798391/112766062-92e0b600-906c-11eb-9d2f-c9f8aa76cc5a.png)

**Comment by vaclavblazek** [comment]
Sorry, unfortunately, I had not found time to investigate yet. Hopefully next week.

**Comment by jrjdavidson** [comment]
Just an update on this- it appears that the issue is caused by how the data is created in metashape. The problem occurs in vts if I generate the tiled model straight from depth maps in metashape. If i create a dense point cloud and then generated the tiled model, it works fine.
cheers

**Comment by vaclavblazek** [comment]
Pls, what OS (distro) and what `vts-tools` version did you use? I tried to convert it myself and there's no broken mesh.

**Comment by jrjdavidson** [comment]
Ubuntu 18.04.5 LTS
slpk2vts 1.89 (built on Jan 29 2020 16:32:20 at bionic)

**Comment by vaclavblazek** [comment]
Tested. Freshly built tag for 1.89 generates broken meshes. Current version is OK. IIRC there were some fixes with properly splitting meshes with too much faces/vertices but I cannot find it in the log.

Anyway, I've made a new `vts-tools` package for bionic, v. 1.93. It's not from the HEAD because there were some very recent updates in metatiles that I do not want to publish into general public yet.
